source/SpinalHDL/Structuring/components_hierarchy.rst

```diff
@@ -56,16 +56,23 @@ The syntax to define inputs and outputs is as follows:
    * - Syntax
      - Description
      - Return
-   * - in Bool()/out Bool()
```

Comment:
Code blocks are broken in this table
...we should really update this to the `in port ...` syntax, but no need to do it in this PR

Reply:
It is how they were before I have not tried to fixup code blocks like that.  Seeing what I can do and how they look after.